src/workerskv/mod.rs

```diff
@@ -19,14 +19,14 @@ pub struct WorkersKVNamespace {
     pub title: String,
 }
 
-impl APIResult for WorkersKVNamespace {}
-impl APIResult for Vec<WorkersKVNamespace> {}
+impl ApiResult for WorkersKVNamespace {}
+impl ApiResult for Vec<WorkersKVNamespace> {}
```

Comment:
Probably also need to do this for `WorkersKvNamespace`.

Reply:
Yes! Good catch

Reply:
@ashleymichal I can't find any occurrences of that in the current master which I've branched off of.

Reply:
i mean `WorkersKVNamespace -> WorkersKvNamespace`, which is present here.

Reply:
Done!